CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# POA Network
+
+## To Contribute
+1. Fork the repository
+https://github.com/poanetwork/hbbft
+2. Create a feature branch
+3. Write tests to cover the work 
+4. Commit changes
+5. Push to the branch
+6. Create a new pull request following the PR protocol below
+
+## Pull Request (PR) Protocol
+
+All pull requests must include: 
+* A clear, readable description of the purpose of the PR
+* A clear, readable description of changes
+* A title that includes (Fix), (Feature), or (Refactor)
```

Comment:
You already broke the rule in this very PR. :stuck_out_tongue: 
I'm still skeptical about it, and my experience with mandatory PR/commit title formats was never great:
* The only projects I found that follow this one are [poa-test-setup](https://github.com/poanetwork/poa-test-setup/commits/master) and [poa-dapps-validators](https://github.com/poanetwork/poa-dapps-validators/commits/core), and even they have already extended the list of allowed keywords ("Update").
* The [RFC](https://github.com/poanetwork/RFC/issues/3) itself is still a draft. Also, it refers only to DApps; hbbft is not a DApp.
* The only way to get a whole team to follow these rules is to check them in CI. But then it's even worse when you have something that doesn't quite fit one of the three categories. This PR here, for example, should really be in a "Docs" category if anything. Although I don't find "(Docs) Readme updates" much more readable than just "Readme updates".

Reply:
Thanks for the feedback. At this stage in the project I think it's appropriate to remove the Contributing section until it is more fleshed out. Once I explore options and get feedback I will create a centralized Contributing file.